registry.json

```diff
@@ -5315,5 +5315,15 @@
    "telegram": "CassandradeVries",
    "operatorName": "",
    "github": "NEVdataDyne"
+ },
+ {
+   "ticker": "PINT",
+   "poolId": "11152917b1e01cb25ac9d993b207bfed11c097b34b77a5ce6f97cfa9",
+   "social": {
+       "twitter": "PINT_token",
+       "website": "https://beerhouse.io/",
```

Comment:
You need to include the `facebook` , `instagram` and `youtube` fields even if blank

Reply:
done